src/components/Monitors/SafeplaceMonitor/utils.ts

```diff
@@ -0,0 +1,46 @@
+export const displayTimetable = (timetable: (string | null)[]): string => {
+    const days = [
+        "Lundi",
+        "Mardi",
+        "Mercredi",
+        "Jeudi",
+        "Vendredi",
+        "Samedi",
+        "Dimanche"
+    ];
+
+    return timetable.map((time, index) => ({ time: time, index: index }))
+        .filter(item => item.time !== null)
+        .map(item => days[item.index] + " : " + item.time)
+        .join(" | ");
+};
+
+export const splitTimetable = (timetable: string): (string | null)[] => {
+    const days = [
```

Comment:
If you want to avoid code repetition you can import your hard coded arrays from an external js file.

example: ` import {days, month, colors} from constantsValues;`

Reply:
I though of it, but I think it is too specific to require an external hard coded array.
Do you think it is needed ?